processing/src/main/java/io/druid/segment/DimensionSelector.java

```diff
@@ -80,10 +90,19 @@
   public String lookupName(int id);
 
   /**
-   * The ID is the int id value of the field.
+   * Returns true if it is possible to {@link #lookupName(int)} by ids from 0 to {@link #getValueCardinality()}
+   * Before the rows with those ids are returns.
    *
-   * @param name field name to look up the id for
-   * @return the id for the given field name
+   * <p>Returns false if {@link #lookupName(int)} could be called only with ids, returned from the last call of {@link
```

Comment:
suggest rewording "last call" and "last IndexedInts" to "most recent call" and "most recent IndexedInts"